### DISCUSSION TITLE
Just want to use plex notify

### DISCUSSION THREAD

**Discussion by Dante2202** [RandomNinjaAtk/arr-scripts#317] [Q&A]
If all I want to use these scripts is for the automatic plex notify do I need to just set everything to false in the “script enablement” section then add my url and token to the plex notify section at the bottom?

**Comment by RandomNinjaAtk** [comment]
Essentially yes, but you need to let auto configure run to add the PlexNotify Script to the ARR app. Or manually add it...

**Reply by Dante2202** [reply]
Perfect thank you! So in all the arr scripts I leave enableAutoConfig=true but can make everything else false? Would I also set everything to false on the ATUOCONFIG SCRIPT SETTINGS section as well except for CustomScripts?

I would like to make use of the cleaner script but if I decided I just wanted the plex notify script how would I just add that to each arr app?

**Reply by RandomNinjaAtk** [reply]
Yes, what you stated should work...

**Comment by Dante2202** [comment]
I added the script and see in "Connect" in radarr that it added the plex notify script but when I added a movie and radarr imported it plex does not get updated with the new movie. The only thing the logs show is that the plexnotity bash script was added.  This is my conf file:
`##### SCRIPT ENABLEMENT
enableAutoConfig="true" # true = enabled :: Enables AutoConfig script to run after startup
enableExtras="false" # true = enabled :: Enables Extras and AutoExtras scripts to run in the background and during import process
enableRecyclarr="false" # true = enabled :: Enables Recyclarr to run
enableQueueCleaner="true" # true = enabled :: Enables QueueCleaner Script that automatically removes stuck downloads that cannot be automatically imported
enableUnmappedFolderCleaner="false" # true = enabled :: Purges any folders that are considered Unmapped in Radarr
enableInvalidMoviesAutoCleaner="false" # true = enabled :: Enables InvalidMoviesAutoCleaner script to run, removes movies that are no longer mapped to TMDB site

##### SCRIPT INTERVALS
autoExtrasScriptInterval=24h #s or #m or #h or #d :: s = seconds, m = minutes, h = hours, d = days :: Amount of time between each script run, when script is enabled
unmappedFolderCleanerScriptInterval=1h #s or #m or #h or #d :: s = seconds, m = minutes, h = hours, d = days :: Amount of time between each script run, when script is enabled
recyclarrScriptInterval=6h #s or #m or #h or #d :: s = seconds, m = minutes, h = hours, d = days :: Amount of time between each script run, when script is enabled
queueCleanerScriptInterval=15m #s or #m or #h or #d :: s = seconds, m = minutes, h = hours, d = days :: Amount of time between each script run, when script is enabled
invalidMoviesAutoCleanerScriptInterval=1h #s or #m or #h or #d :: s = seconds, m = minutes, h = hours, d = days :: Amount of time between each script run, when script is enabled

##### ATUOCONFIG SCRIPT SETTINGS
configureMediaManagement="false"
configureMetadataProviderSettings="false"
configureCustomScripts="true"
configureCustomFormats="false"
configureNaming="false"

##### EXTRAS SCRIPT
extrasLanguages="en-US" # Set the desired language for Extras, all languages will be processed... (this is a "," separated list of TMDB language codes, get the code from there sites language opitons, example: en-US)
extrasType="all" # all or trailers :: all downloads all available videos (trailers, clips, featurette, etc...) :: trailers only downloads trailers
extrasOfficialOnly="false" # true = enabled :: Skips extras that are not considered/marked as Official from TMDB site.
extrasKodiCompatibility="false" # true = enabled :: Only works if "extrasSingle" is set to true, names trailer in a kodi compatible naming scheme (movie-trailer.mkv)
extrasSingle="false" # true = enabled :: Only downloads the first available trailer, does not download any other extras
videoFormat="bestvideo*+bestaudio/best" # OPTIONAL - yt-dlp video selection paramater, do not change unless you know what your doing....

##### RECYCLARR SCRIPT
recyclarrConfig="/config/extended/recyclarr.yaml" # Change to a custom yaml file to use your own configuration, the default file is always overwritten...

##### PLEX NOTIFY SCRIPT
plexUrl="REDACTED" # ONLY used if PlexNotify.bash is used, example: http://x.x.x.x:32400
plexToken="REDACTED" # ONLY used if PlexNotify.bash is used

##### TDARR SCAN SCRIPT
tdarrUrl="" # ONLY used if TdarrScan.bash is used, example: http://x.x.x.x:8265
tdarrDbID="" # ONLY used if TdarrScan.bash is used. The ID of the library you want to scan in Tdarr
`

**Reply by RandomNinjaAtk** [reply]
You need to look at the PlexNotify.txt log file to see what is happening. If that log file does not exist, then radarr is not triggering the script...
